### PR TITLE
Fix TestRBACV1Group and TestRBACV2Group

### PR DESCRIPTION
**Pull request by yangminzhu**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[x] Security
[x] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

The test is broken by https://github.com/istio/istio/pull/15224, not sure why it passed the tests and get merged in the first place. I filed https://github.com/istio/istio/issues/15315 for this issue.

### CODE REVIEW

File: tests/common/jwt/jwt_token_test.go [hunk 1]
```diff
@@ -106,28 +107,13 @@ func TestSampleJwtToken(t *testing.T) {
...
-			switch v.(type) {
-			case string:
-				got, ok := claims[k].(string)
-				if ok {
-					if got != v {
-						t.Errorf("%s: claim %q got value %q but want %q", tc.name, k, got, v)
-					}
-				} else {
-					t.Errorf("%s: claim %q got type %T but want string", tc.name, k, claims[k])
+			got, ok := claims[k].(interface{})
```
Comment (howardjohn) [inline tests/common/jwt/jwt_token_test.go]: isn't `ok` here checking that the cast to `.(interface{})` succeeded? 

Also seems the cast is unneeded, isn't it already an `interface{}`?

**Comment by howardjohn** [comment]
lint will be fixed by https://github.com/istio/test-infra/pull/1414

File: tests/common/jwt/jwt_token_test.go (hunk 1 above)
Comment (yangminzhu) [reply tests/common/jwt/jwt_token_test.go]: Ah, you're right, the cast is unneeded. The `ok` is checking if the key is present in the `claims`.

Comment (howardjohn) [reply tests/common/jwt/jwt_token_test.go]: Both a map lookup and cast return a second `ok` value. Looks like the end value is essentially that they both succeed https://play.golang.org/p/GijhZ5UT0jk. Learned something new today :slightly_smiling_face:

**Comment by howardjohn** [comment]
/test istio-lint-master
linter fix should be merged

**Comment by yangminzhu** [comment]
/hold 
I'll check manually to make sure the `prow/integ-security-k8s-presubmit-tests.sh` pass in this PR

**Comment by howardjohn** [comment]
```
tests/common/jwt/jwt_token_test.go:58: File is not `gofmt`-ed with `-s` (gofmt)
```

**Comment by howardjohn** [comment]
but looks like it does fix security

**Comment by yangminzhu** [comment]
@howardjohn Yes, I think the test is fixed in this PR.

**Pull request merged**